### PR TITLE
fix(embeddings): fix azure embeddings

### PR DESCRIPTION
**Pull request by perezpefaur**
Fix [issue](https://github.com/hwchase17/langchainjs/issues/1850) with azure open AI embeddings to be consistent with [langchain page](https://js.langchain.com/docs/modules/data_connection/text_embedding/integrations/azure_openai). And sorted the imports.

### CODE REVIEW

**Comment by jacoblee93** [comment]
Thanks for the PR! What's the issue here exactly? I don't think I quite understood from the link.

The base path should include `openai/deployments` by default: https://github.com/hwchase17/langchainjs/blob/main/langchain/src/util/azure.ts#L59

File: langchain/src/embeddings/openai.ts [hunk 1]
```diff
@@ -106,7 +106,12 @@ export class OpenAIEmbeddings
     this.azureOpenAIApiVersion = azureApiVersion;
     this.azureOpenAIApiKey = azureApiKey;
     this.azureOpenAIApiInstanceName = azureApiInstanceName;
-    this.azureOpenAIApiDeploymentName = azureApiDeploymentName;
+    // if openai/deployments is not in the deployment name, add it (for backwards compatibility)
+    if (azureApiDeploymentName?.indexOf("openai/deployments") === -1) {
+      this.azureOpenAIApiDeploymentName = `openai/deployments/${azureApiDeploymentName}`;
```
Comment (pavelpiha) [inline langchain/src/embeddings/openai.ts:111]: I don't think this will fix. please look to my ticket https://github.com/hwchase17/langchainjs/issues/2105

**Comment by nimal-outreach** [comment]
> Thanks for the PR! What's the issue here exactly? I don't think I quite understood from the link.
> 
> The base path should include `openai/deployments` by default: https://github.com/hwchase17/langchainjs/blob/main/langchain/src/util/azure.ts#L59

If we would provide the API key, base path, and deployment name (all of which are documented on [langchain page](https://js.langchain.com/docs/modules/data_connection/text_embedding/integrations/azure_openai)) that line will not be executed at all


  if (
    azureOpenAIApiKey &&
    azureOpenAIBasePath &&
    azureOpenAIApiDeploymentName
  ) {
    return `${azureOpenAIBasePath}/${azureOpenAIApiDeploymentName}`;
  }

**Comment by jacoblee93** [comment]
We've clarified the page here in #2318 - hope that helps make things clearer:

https://js.langchain.com/docs/modules/data_connection/text_embedding/integrations/azure_openai

**Pull request closed without merging**